src/Behat/Behat/Output/Node/Printer/JUnit/JUnitScenarioPrinter.php

```diff
@@ -76,12 +76,63 @@ public function printOpenTag(Formatter $formatter, FeatureNode $feature, Scenari
         /** @var JUnitOutputPrinter $outputPrinter */
         $outputPrinter = $formatter->getOutputPrinter();
 
-        $outputPrinter->addTestcase(array(
-            'name' => $name,
+        $outputPrinter->addTestcase($this->addCircleCiAttributes(array(
+            'name' => $this->addLineNumberToScenarioName($name, $scenario->getLine()),
+            'file' => $this->addLineNumberToFeatureFile($this->convertToRelativePath($feature->getFile()), $scenario->getLine()),
             'classname' => $feature->getTitle(),
             'status' => $this->resultConverter->convertResultToString($result),
             'time' => $this->durationListener ? $this->durationListener->getDuration($scenario) : ''
-        ));
+        )));
+    }
+
+    /**
+     * Append scenario line number in the feature file to a scenario name
+     *
+     * @param string $name
+     * @param integer $lineNumber
+     * @return string
+     */
+    private function addLineNumberToScenarioName(string $name, int $lineNumber) {
+        return $name . " Line #" . $lineNumber;
```

Comment:
Double quotes are necessary if you don't need string interpolation.
But I feel like if we use string interpolation, we can actually simplify this:

```suggestion
        return "$name Line #$lineNumber";
```

Please do this for similar string concatenation in this PR.